doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1181,7 +1345,7 @@ Pattern matching on goals and hypotheses: match goal
       first to last.  :ref:`Normal<match_goal_hyps_ex>` and :ref:`reverse<match_goal_hyps_rev_ex>` examples.
 
    :n:`|- @match_pattern`
-      A pattern to match with the current goal
+      A pattern to match with the current goals
```

Comment:
Why this change?

Reply:
Because I expect if you have `split; match goal ...` it processes each goal independently like the other constructs.  (Though a little tedious to repeat that everywhere.)

Reply:
Indeed, `match` processes each goal independently.  I think we're on the same page about semantics, but on different pages about text interpretation; to me, "A pattern to match with the current goals" suggests/implies "this single pattern will be matched against all current goals; if it matches all of them, the branch triggers (once, on all goals), while if it fails to match any of them, it fails to match overall".  The "process each goal independently" should go to the `match` statement as a whole, I believe; by the time the code is looking at `@match_pattern`, there is only one goal under focus / only one current goal.